src/ralph/data_importer/management/commands/importer.py

```diff
@@ -79,8 +83,49 @@ def add_arguments(self, parser):
             default='utf-8',
             help="Output encoding",
         )
+        parser.add_argument(
+            '--dir',
+            help="Csv files directory",
+        )
+        parser.add_argument(
+            '--zipfile',
+            help="ZIP File name",
+        )
+
+    def from_zip(self, options):
+        with open(options.get('zipfile'), 'rb') as f:
+            out_path = tempfile.mkdtemp()
+            z = zipfile.ZipFile(f)
+            for name in z.namelist():
+                z.extract(name, out_path)
+            options['dir'] = out_path
+            options['skipid'] = True
```

Comment:
When importing from old ralph - yes. But what when someone want to import everything without skipping? IMO this flag should stay as it is and when importing from NG we should use it explicitly (`--skipid`)